hack/e2e-common.sh

```diff
@@ -221,8 +222,27 @@ function cluster_kueue_deploy {
         kubectl -n cert-manager wait --for condition=ready pod \
             -l app.kubernetes.io/instance=cert-manager \
             --timeout=5m
-        
-        deploy_with_certmanager "$1"
+        if [ "$E2E_USE_HELM" == 'true' ]; then
+            $HELM install \
+              -f "${ROOT_DIR}/test/e2e/config/certmanager/values.yaml" \
```

Comment:
Can we eliminate this duplication? Ideally when using `CERTMANAGER_VERSION` we point to a different values.yaml file, but the commands should be the same.